install/install_for_macos_linux.sh

```diff
@@ -14,11 +14,13 @@ warning() {
     DISTRO_NAME=$(cat /etc/*elease | grep DISTRIB_ID | cut -d '=' -f2)
     DISTRO_VERSION=$(cat /etc/*elease | grep DISTRIB_RELEASE | cut -d '=' -f2 | sed 's@^[^0-9]*\([0-9]\+\).*@\1@')
     if [[ -z "$DISTRO_VERSION" ]] || [[ -z "$DISTRO_NAME" ]] || ( [ "$DISTRO_VERSION" -lt 20 ] && [ "$DISTRO_NAME" != "Ubuntu" ]);then
-      warning "tipi is designed to work with at least Ubuntu 20.04"
-      warning "But would you still like to try to install tipi (y/n) ?"
-      read answer
-      if [ "$answer" == "${answer#[Yy]}" ] ;then 
-       exit 1
+      if [ ! -f "/etc/arch-release" ]; then
```

Comment:
I don't like that. Either we support only ubuntu or arch and ubuntu officially. In both cases that check is ugly. (like, why is it even nested?)